easybuild/tools/module_naming_scheme/mns.py

```diff
@@ -143,7 +143,7 @@ def det_init_modulepaths(self, ec):
         """
         return []
 
-    def expand_toolchain_load(self):
+    def expand_toolchain_load(self, ec):
```

Comment:
@bartoldeman This is not backward-compatible as is, it would break existing custom module naming scheme implementations.

However, with a bit of extra care we can make it backward-compatible:

```python
def expand_toolchain_load(self, ec=None):
```